<!DOCTYPE html>
<html amp lang="en-US" i-amphtml-layout="" i-amphtml-no-boilerplate="" transformed="self;v=1">
<head><meta charset="utf-8"><style amp-runtime="" i-amphtml-version="012111242025001">html{overflow-x:hidden!important}html.i-amphtml-fie{height:100%!important;width:100%!important}html:not([amp4ads]),html:not([amp4ads]) body{height:auto!important}html:not([amp4ads]) body{margin:0!important}body{-webkit-text-size-adjust:100%;-moz-text-size-adjust:100%;-ms-text-size-adjust:100%;text-size-adjust:100%}html.i-amphtml-singledoc.i-amphtml-embedded{-ms-touch-action:pan-y pinch-zoom;touch-action:pan-y pinch-zoom}html.i-amphtml-fie>body,html.i-amphtml-singledoc>body{overflow:visible!important}html.i-amphtml-fie:not(.i-amphtml-inabox)>body,html.i-amphtml-singledoc:not(.i-amphtml-inabox)>body{position:relative!important}html.i-amphtml-ios-embed-legacy>body{overflow-x:hidden!important;overflow-y:auto!important;position:absolute!important}html.i-amphtml-ios-embed{overflow-y:auto!important;position:static}#i-amphtml-wrapper{overflow-x:hidden!important;overflow-y:auto!important;position:absolute!important;top:0!important;left:0!important;right:0!important;bottom:0!important;margin:0!important;display:block!important}html.i-amphtml-ios-embed.i-amphtml-ios-overscroll,html.i-amphtml-ios-embed.i-amphtml-ios-overscroll>#i-amphtml-wrapper{-webkit-overflow-scrolling:touch!important}#i-amphtml-wrapper>body{position:relative!important;border-top:1px solid transparent!important}#i-amphtml-wrapper+body{visibility:visible}#i-amphtml-wrapper+body .i-amphtml-lightbox-element,#i-amphtml-wrapper+body[i-amphtml-lightbox]{visibility:hidden}#i-amphtml-wrapper+body[i-amphtml-lightbox] .i-amphtml-lightbox-element{visibility:visible}#i-amphtml-wrapper.i-amphtml-scroll-disabled,.i-amphtml-scroll-disabled{overflow-x:hidden!important;overflow-y:hidden!important}amp-instagram{padding:54px 0px 0px!important;background-color:#fff}amp-iframe iframe{box-sizing:border-box!important}[amp-access][amp-access-hide]{display:none}[subscriptions-dialog],body:not(.i-amphtml-subs-ready) [subscriptions-action],body:not(.i-amphtml-subs-ready) [subscriptions-section]{display:none!important}amp-experiment,amp-live-list>[update]{display:none}amp-list[resizable-children]>.i-amphtml-loading-container.amp-hidden{display:none!important}amp-list [fetch-error],amp-list[load-more] [load-more-button],amp-list[load-more] [load-more-end],amp-list[load-more] [load-more-failed],amp-list[load-more] [load-more-loading]{display:none}amp-list[diffable] div[role=list]{display:block}amp-story-page,amp-story[standalone]{min-height:1px!important;display:block!important;height:100%!important;margin:0!important;padding:0!important;overflow:hidden!important;width:100%!important}amp-story[standalone]{background-color:#000!important;position:relative!important}amp-story-page{background-color:#757575}amp-story .amp-active>div,amp-story .i-amphtml-loader-background{display:none!important}amp-story-page:not(:first-of-type):not([distance]):not([active]){transform:translateY(1000vh)!important}amp-autocomplete{position:relative!important;display:inline-block!important}amp-autocomplete>input,amp-autocomplete>textarea{padding:0.5rem;border:1px solid rgba(0,0,0,0.33)}.i-amphtml-autocomplete-results,amp-autocomplete>input,amp-autocomplete>textarea{font-size:1rem;line-height:1.5rem}[amp-fx^=fly-in]{visibility:hidden}amp-script[nodom],amp-script[sandboxed]{position:fixed!important;top:0!important;width:1px!important;height:1px!important;overflow:hidden!important;visibility:hidden}
/*# sourceURL=/css/ampdoc.css*/[hidden]{display:none!important}.i-amphtml-element{display:inline-block}.i-amphtml-blurry-placeholder{transition:opacity 0.3s cubic-bezier(0.0,0.0,0.2,1)!important;pointer-events:none}[layout=nodisplay]:not(.i-amphtml-element){display:none!important}.i-amphtml-layout-fixed,[layout=fixed][width][height]:not(.i-amphtml-layout-fixed){display:inline-block;position:relative}.i-amphtml-layout-responsive,[layout=responsive][width][height]:not(.i-amphtml-layout-responsive),[width][height][heights]:not([layout]):not(.i-amphtml-layout-responsive),[width][height][sizes]:not(img):not([layout]):not(.i-amphtml-layout-responsive){display:block;position:relative}.i-amphtml-layout-intrinsic,[layout=intrinsic][width][height]:not(.i-amphtml-layout-intrinsic){display:inline-block;position:relative;max-width:100%}.i-amphtml-layout-intrinsic .i-amphtml-sizer{max-width:100%}.i-amphtml-intrinsic-sizer{max-width:100%;display:block!important}.i-amphtml-layout-container,.i-amphtml-layout-fixed-height,[layout=container],[layout=fixed-height][height]:not(.i-amphtml-layout-fixed-height){display:block;position:relative}.i-amphtml-layout-fill,.i-amphtml-layout-fill.i-amphtml-notbuilt,[layout=fill]:not(.i-amphtml-layout-fill),body noscript>*{display:block;overflow:hidden!important;position:absolute;top:0;left:0;bottom:0;right:0}body noscript>*{position:absolute!important;width:100%;height:100%;z-index:2}body noscript{display:inline!important}.i-amphtml-layout-flex-item,[layout=flex-item]:not(.i-amphtml-layout-flex-item){display:block;position:relative;-ms-flex:1 1 auto;flex:1 1 auto}.i-amphtml-layout-fluid{position:relative}.i-amphtml-layout-size-defined{overflow:hidden!important}.i-amphtml-layout-awaiting-size{position:absolute!important;top:auto!important;bottom:auto!important}i-amphtml-sizer{display:block!important}@supports (aspect-ratio:1/1){i-amphtml-sizer.i-amphtml-disable-ar{display:none!important}}.i-amphtml-blurry-placeholder,.i-amphtml-fill-content{display:block;height:0;max-height:100%;max-width:100%;min-height:100%;min-width:100%;width:0;margin:auto}.i-amphtml-layout-size-defined .i-amphtml-fill-content{position:absolute;top:0;left:0;bottom:0;right:0}.i-amphtml-replaced-content,.i-amphtml-screen-reader{padding:0!important;border:none!important}.i-amphtml-screen-reader{position:fixed!important;top:0px!important;left:0px!important;width:4px!important;height:4px!important;opacity:0!important;overflow:hidden!important;margin:0!important;display:block!important;visibility:visible!important}.i-amphtml-screen-reader~.i-amphtml-screen-reader{left:8px!important}.i-amphtml-screen-reader~.i-amphtml-screen-reader~.i-amphtml-screen-reader{left:12px!important}.i-amphtml-screen-reader~.i-amphtml-screen-reader~.i-amphtml-screen-reader~.i-amphtml-screen-reader{left:16px!important}.i-amphtml-unresolved{position:relative;overflow:hidden!important}.i-amphtml-select-disabled{-webkit-user-select:none!important;-ms-user-select:none!important;user-select:none!important}.i-amphtml-notbuilt,[layout]:not(.i-amphtml-element),[width][height][heights]:not([layout]):not(.i-amphtml-element),[width][height][sizes]:not(img):not([layout]):not(.i-amphtml-element){position:relative;overflow:hidden!important;color:transparent!important}.i-amphtml-notbuilt:not(.i-amphtml-layout-container)>*,[layout]:not([layout=container]):not(.i-amphtml-element)>*,[width][height][heights]:not([layout]):not(.i-amphtml-element)>*,[width][height][sizes]:not([layout]):not(.i-amphtml-element)>*{display:none}amp-img:not(.i-amphtml-element)[i-amphtml-ssr]>img.i-amphtml-fill-content{display:block}.i-amphtml-notbuilt:not(.i-amphtml-layout-container),[layout]:not([layout=container]):not(.i-amphtml-element),[width][height][heights]:not([layout]):not(.i-amphtml-element),[width][height][sizes]:not(img):not([layout]):not(.i-amphtml-element){color:transparent!important;line-height:0!important}.i-amphtml-ghost{visibility:hidden!important}.i-amphtml-element>[placeholder],[layout]:not(.i-amphtml-element)>[placeholder],[width][height][heights]:not([layout]):not(.i-amphtml-element)>[placeholder],[width][height][sizes]:not([layout]):not(.i-amphtml-element)>[placeholder]{display:block;line-height:normal}.i-amphtml-element>[placeholder].amp-hidden,.i-amphtml-element>[placeholder].hidden{visibility:hidden}.i-amphtml-element:not(.amp-notsupported)>[fallback],.i-amphtml-layout-container>[placeholder].amp-hidden,.i-amphtml-layout-container>[placeholder].hidden{display:none}.i-amphtml-layout-size-defined>[fallback],.i-amphtml-layout-size-defined>[placeholder]{position:absolute!important;top:0!important;left:0!important;right:0!important;bottom:0!important;z-index:1}amp-img[i-amphtml-ssr]:not(.i-amphtml-element)>[placeholder]{z-index:auto}.i-amphtml-notbuilt>[placeholder]{display:block!important}.i-amphtml-hidden-by-media-query{display:none!important}.i-amphtml-element-error{background:red!important;color:#fff!important;position:relative!important}.i-amphtml-element-error:before{content:attr(error-message)}i-amp-scroll-container,i-amphtml-scroll-container{position:absolute;top:0;left:0;right:0;bottom:0;display:block}i-amp-scroll-container.amp-active,i-amphtml-scroll-container.amp-active{overflow:auto;-webkit-overflow-scrolling:touch}.i-amphtml-loading-container{display:block!important;pointer-events:none;z-index:1}.i-amphtml-notbuilt>.i-amphtml-loading-container{display:block!important}.i-amphtml-loading-container.amp-hidden{visibility:hidden}.i-amphtml-element>[overflow]{cursor:pointer;position:relative;z-index:2;visibility:hidden;display:initial;line-height:normal}.i-amphtml-layout-size-defined>[overflow]{position:absolute}.i-amphtml-element>[overflow].amp-visible{visibility:visible}template{display:none!important}.amp-border-box,.amp-border-box *,.amp-border-box :after,.amp-border-box :before{box-sizing:border-box}amp-pixel{display:none!important}amp-analytics,amp-auto-ads,amp-story-auto-ads{position:fixed!important;top:0!important;width:1px!important;height:1px!important;overflow:hidden!important;visibility:hidden}html.i-amphtml-fie>amp-analytics{position:initial!important}[visible-when-invalid]:not(.visible),form [submit-error],form [submit-success],form [submitting]{display:none}amp-accordion{display:block!important}@media (min-width:1px){:where(amp-accordion>section)>:first-child{margin:0;background-color:#efefef;padding-right:20px;border:1px solid #dfdfdf}:where(amp-accordion>section)>:last-child{margin:0}}amp-accordion>section{float:none!important}amp-accordion>section>*{float:none!important;display:block!important;overflow:hidden!important;position:relative!important}amp-accordion,amp-accordion>section{margin:0}amp-accordion:not(.i-amphtml-built)>section>:last-child{display:none!important}amp-accordion:not(.i-amphtml-built)>section[expanded]>:last-child{display:block!important}
/*# sourceURL=/css/ampshared.css*/</style><meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1"><title>安装Bioperl最基本模块Bio::SeqIO - 有组织在!</title><link rel="preconnect" href="https://cdn.ampproject.org"><link rel="preload" as="script" href="https://cdn.ampproject.org/v0.js"><script async="" src="https://cdn.ampproject.org/v0.js"></script><style amp-custom="">html{background:#0a89c0}body{background:#fff;color:#353535;font-family:Georgia,"Times New Roman",Times,Serif;font-weight:300;line-height:1.75em}p{margin:0 0 1em;padding:0}a,a:visited{color:#0a89c0}a:hover,a:active,a:focus{color:#353535}.amp-wp-meta,.amp-wp-header div,.amp-wp-title,.amp-wp-tax-category,.amp-wp-footer p,.back-to-top{font-family:-apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen-Sans","Ubuntu","Cantarell","Helvetica Neue",sans-serif}.amp-wp-header{background-color:#0a89c0}.amp-wp-header div{color:#fff;font-size:1em;font-weight:400;margin:0 auto;max-width:calc(840px - 32px);padding:.875em 16px;position:relative}.amp-wp-header a{color:#fff;text-decoration:none}.amp-wp-header .amp-wp-site-icon{background-color:#fff;border:1px solid #fff;border-radius:50%;position:absolute;right:18px;top:10px}.amp-wp-article{color:#353535;font-weight:400;margin:1.5em auto;max-width:840px;overflow-wrap:break-word;word-wrap:break-word}.amp-wp-article-header{align-items:center;align-content:stretch;display:flex;flex-wrap:wrap;justify-content:space-between;margin:1.5em 16px 0}.amp-wp-title{color:#353535;display:block;flex:1 0 100%;font-weight:900;margin:0 0 .625em;width:100%}.amp-wp-meta{color:#696969;display:inline-block;flex:2 1 50%;font-size:.875em;line-height:1.5em;margin:0 0 1.5em;padding:0}.amp-wp-article-header .amp-wp-meta:last-of-type{text-align:right}.amp-wp-article-header .amp-wp-meta:first-of-type{text-align:left}.amp-wp-byline amp-img,.amp-wp-byline .amp-wp-author{display:inline-block;vertical-align:middle}.amp-wp-byline amp-img{border:1px solid #0a89c0;border-radius:50%;position:relative;margin-right:6px}.amp-wp-posted-on{text-align:right}.amp-wp-article-content{margin:0 16px}.amp-wp-article-content amp-img{margin:0 auto}.amp-wp-article-footer .amp-wp-meta{display:block}.amp-wp-tax-category{color:#696969;font-size:.875em;line-height:1.5em;margin:1.5em 16px}.amp-wp-footer{border-top:1px solid #c2c2c2;margin:calc(1.5em - 1px) 0 0}.amp-wp-footer div{margin:0 auto;max-width:calc(840px - 32px);padding:1.25em 16px 1.25em;position:relative}.amp-wp-footer h2{font-size:1em;line-height:1.375em;margin:0 0 .5em}.amp-wp-footer p{color:#696969;font-size:.8em;line-height:1.5em;margin:0 85px 0 0}.amp-wp-footer a{text-decoration:none}.back-to-top{bottom:1.275em;font-size:.8em;font-weight:600;line-height:2em;position:absolute;right:16px}.htmledit_views span[lang]{font-style:italic}.htmledit_views{font-family:-apple-system,SF UI Text,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif,SimHei,SimSun}.htmledit_views a>amp-img{padding:1px;margin:1px;border:none;outline:#0782c1 solid 1px}.htmledit_views p{font-size:16px;color:#4d4d4d;font-weight:400;line-height:26px;margin:0 0 16px;overflow-x:auto}.htmledit_views amp-img{max-width:100%}.htmledit_views *{box-sizing:border-box}.htmledit_views h1,.htmledit_views h2{color:#4f4f4f;margin:8px 0 16px;font-weight:700}.htmledit_views h1{font-size:28px;line-height:36px}.htmledit_views h2{font-size:24px;line-height:32px}.htmledit_views a{color:#4ea1db;text-decoration:none}.htmledit_views a:focus,.htmledit_views a:hover{color:#ca0c16}.htmledit_views a:visited{color:#6795b5}

/*# sourceURL=amp-custom.css */</style><script type="application/ld+json" class="yoast-schema-graph yoast-schema-graph--main">{"@context":"https://schema.org","@graph":[{"@type":"WebSite","@id":"https://uzzz.org/#website","url":"https://uzzz.org/","name":"\u6709\u7ec4\u7ec7\u5728!","potentialAction":{"@type":"SearchAction","target":"https://uzzz.org/?s={search_term_string}","query-input":"required name=search_term_string"}},{"@type":"WebPage","@id":"https://uzzz.org/article/3030/#webpage","url":"https://uzzz.org/article/3030/","inLanguage":"en-US","name":"\u5b89\u88c5Bioperl\u6700\u57fa\u672c\u6a21\u5757Bio::SeqIO - \u6709\u7ec4\u7ec7\u5728!","isPartOf":{"@id":"https://uzzz.org/#website"},"datePublished":"2016-02-27T09:50:09+00:00","dateModified":"2016-02-27T09:50:09+00:00","author":{"@id":"https://uzzz.org/#/schema/person/29673f1347b0abda5882803c72ee5a3f"}},{"@type":["Person"],"@id":"https://uzzz.org/#/schema/person/29673f1347b0abda5882803c72ee5a3f","name":"fandyvon","sameAs":[]}]}</script><link rel="canonical" href="https://uzzz.org/article/3030/"></head>

<body class="">

<header id="top" class="amp-wp-header">
	<div>
		<a href="https://uzzz.org/">
										<amp-img src="https://uzzz.org/wp-content/uploads/2019/10/cropped-icon-32x32.png" width="32" height="32" class="amp-wp-site-icon i-amphtml-layout-fixed i-amphtml-layout-size-defined" style="width:32px;height:32px;" i-amphtml-layout="fixed"></amp-img>
						<span class="amp-site-title">
				有组织在!			</span>
		</a>

					</div>
</header>

<article class="amp-wp-article">
	<header class="amp-wp-article-header">
		<h1 class="amp-wp-title">安装Bioperl最基本模块Bio::SeqIO</h1>
			<div class="amp-wp-meta amp-wp-byline">
					<amp-img src="https://secure.gravatar.com/avatar/e786821a74ef0467825a7d60183307bc?s=24&d=mm&r=g" alt="fandyvon" width="24" height="24" layout="fixed" class="i-amphtml-layout-fixed i-amphtml-layout-size-defined" style="width:24px;height:24px;" i-amphtml-layout="fixed"></amp-img>
				<span class="amp-wp-author author vcard">fandyvon</span>
	</div>
<div class="amp-wp-meta amp-wp-posted-on">
	<time datetime="2016-02-27T17:50:09+00:00">
		6 years ago	</time>
</div>
	</header>

	
	<div class="amp-wp-article-content">
		<div id="article_content" class="article_content clearfix">
 <br>
 
 
 
<div class="htmledit_views" id="content_views">
<div>
   Bioperl就像个大柜子，里面存放各种模块。Bio::SeqIO是Bioperl最基本的模块，本文介绍其在CentOS7-64bit下的安装方法。
  </div>
<div>
   
  </div>
<div>
   STEP1：prepare to install
  </div>
<div>
   
  </div>
<div>
   yum update #更新yum源。
  </div>
<div>
   
  </div>
<div>
   yum install expat-devel 包  #Bioperl官网说要用XML功能需要安装expat和expat-devel包，由于expat包CentOS7自带，所以安装expat-devel包，究竟是不是必不可少的还需验证。
  </div>
<div>
   
  </div>
<div>
   yum install ruby   #Gudiance说要安装ruby，暂且记下
  </div>
<div>
   
  </div>
<div>
   yum install cpan #安装perl的cpan包
  </div>
<div>
   
  </div>
<div>
   cpan  #初次运行cpan需要设置，一路选择默认设置即可。
  </div>
<div>
   
  </div>
<div>
   cpan>install Bundle::CPAN  #更新CPAN
  </div>
<div>
   
  </div>
<div>
   
  </div>
<div>
   
  </div>
<div>
   <br> cpan>install Module::Build   #安装Module::Build 模块并设置<br>
   <br> cpan>o conf prefer_installer MB<br>
   <br> cpan>o conf commit
  </div>
<div>
   
  </div>
<div>
   cpan>install LWP  #安装LWP，这步很重要，不然Bio::SeqIO编译不成功。
  </div>
<div>
   
  </div>
<div>
   cpan>install Digest::MD5  #安装或更新Digest::MD5，否则Bio::SeqIO编译不成功。
  </div>
<div>
   
  </div>
<div>
   
  </div>
<div>
   
  </div>
<div>
   STEP2: Install Bio::SeqIO
  </div>
<div>
   
  </div>
<div>
   cpan>install Bio::SeqIO #安装Bioperl最基本的模块Bio::SeqIO
  </div>
<div>
   
  </div>
<div>
   #以下是CPAN对此包的依赖性检查，其中必须要的模块cpan会自动安装，推荐模块如在使用中遇到问题请安装此模块。
  </div>
<div>
<div>
    Reading ‘/root/.cpan/Metadata’<br>
    <br>   Database was generated on Fri, 26 Feb 2016 12:17:02 GMT<br>
    <br> Running install for module ‘Bio::SeqIO’<br>
    <br> Fetching with LWP:<br>
    <br> http://cpan.develooper.com/authors/id/C/CJ/CJFIELDS/BioPerl-1.6.924.tar.gz<br>
    <br> Fetching with LWP:<br>
    <br> http://cpan.develooper.com/authors/id/C/CJ/CJFIELDS/CHECKSUMS<br>
    <br> Checksum for /root/.cpan/sources/authors/id/C/CJ/CJFIELDS/BioPerl-1.6.924.tar.gz ok<br>
    <br> Scanning cache /root/.cpan/build for sizes<br>
    <br> ………………………………………………………………….DONE<br>
    <br> Configuring C/CJ/CJFIELDS/BioPerl-1.6.924.tar.gz with Build.PL<br>
    <br> Checking prerequisites…<br>
    <br>   requires:<br>
    <br>     !  Data::Stag is not installed<br>
    <br>     !  IO::String is not installed<br>
    <br>   build_requires:<br>
    <br>     !  Test::Most is not installed<br>
    <br>   recommends:<br>
    <br>     *  Algorithm::Munkres is not installed<br>
    <br>     *  Array::Compare is not installed<br>
    <br>     *  Bio::Phylo is not installed<br>
    <br>     *  Clone is not installed<br>
    <br>     *  Convert::Binary::C is not installed<br>
    <br>     *  DB_File is not installed<br>
    <br>     *  GD is not installed<br>
    <br>     *  Graph is not installed<br>
    <br>     *  GraphViz is not installed<br>
    <br>     *  HTML::TableExtract is not installed<br>
    <br>     *  IO::Scalar is not installed<br>
    <br>     *  List::MoreUtils is not installed<br>
    <br>     *  PostScript::TextBlock is not installed<br>
    <br>     *  SOAP::Lite is not installed<br>
    <br>     *  SVG is not installed<br>
    <br>     *  SVG::Graph is not installed<br>
    <br>     *  Set::Scalar is not installed<br>
    <br>     *  Sort::Naturally is not installed<br>
    <br>     *  Spreadsheet::ParseExcel is not installed<br>
    <br>     *  XML::DOM is not installed<br>
    <br>     *  XML::DOM::XPath is not installed<br>
    <br>     *  XML::LibXML is not installed<br>
    <br>     *  XML::Parser::PerlSAX is not installed<br>
    <br>     *  XML::SAX is not installed<br>
    <br>     *  XML::SAX::Writer is not installed<br>
    <br>     *  XML::Simple is not installed<br>
    <br>     *  XML::Twig is not installed<br>
    <br>     *  XML::Writer is not installed
   </div>
<div>
    
   </div>
<div>
    Checking optional features…<br>
    <br> Bio::DB::GFF Tests….disabled<br>
    <br>   requires:<br>
    <br>     ! DBI is not installed<br>
    <br> DB_File Tests………disabled<br>
    <br>   requires:<br>
    <br>     ! DB_File is not installed<br>
    <br> EntrezGene…………disabled<br>
    <br>   requires:<br>
    <br>     ! Bio::ASN1::EntrezGene is not installed<br>
    <br> MySQL Tests………..disabled<br>
    <br>   requires:<br>
    <br>     ! DBD::mysql is not installed<br>
    <br>     ! DBI is not installed<br>
    <br> Pg Tests…………..disabled<br>
    <br>   requires:<br>
    <br>     ! DBD::Pg is not installed<br>
    <br>     ! DBI is not installed<br>
    <br> SQLite Tests……….disabled<br>
    <br>   requires:<br>
    <br>     ! DBD::SQLite is not installed<br>
    <br>     ! DBI is not installed
   </div>
<div>
    
   </div>
<div>
    ERRORS/WARNINGS FOUND IN PREREQUISITES.  You may wish to install the versions<br>
    <br> of the modules indicated above before proceeding with this installation<br>
    
   </div>
<p>
  </p></div>
<p>附件中有我第一次安装Bio::SeqIO时完成的test文档，哪个包需要哪些依赖包请参考。</p>

<div>
    #从这里开始正式要装Bio::Perl了<br>
   <br>   CJFIELDS/BioPerl-1.6.924.tar.gz<br>
   <br>   Has already been unwrapped into directory /root/.cpan/build/BioPerl-1.6.924-4qo47i<br>
   <br>   CJFIELDS/BioPerl-1.6.924.tar.gz<br>
   <br>   Has already been prepared<br>
   <br> Running Build for C/CJ/CJFIELDS/BioPerl-1.6.924.tar.gz<br>
   <br> Building BioPerl<br>
   <br>   CJFIELDS/BioPerl-1.6.924.tar.gz<br>
   <br>   ./Build — OK<br>
   <br> Running Build test<br>
   <br> t/Align/AlignStats.t ………………. ok<br>
   <br> t/Align/AlignUtil.t ……………….. ok<br>
   <br> t/Align/Graphics.t ………………… skipped: The optional module GD (or dependencies thereof) was not installed<br>
   <br> t/Align/SimpleAlign.t ……………… ok<br>
   <br> t/Align/TreeBuild.t ……………….. ok<br>
   <br> t/Align/Utilities.t ……………….. ok<br>
   <br> t/AlignIO/AlignIO.t ……………….. ok<br>
   <br> t/AlignIO/arp.t …………………… ok<br>
   <br> t/AlignIO/bl2seq.t ………………… ok<br>
   <br> t/AlignIO/clustalw.t ………………. ok<br>
   <br> t/AlignIO/emboss.t ………………… ok<br>
   <br> t/AlignIO/fasta.t …………………. ok<br>
   <br> t/AlignIO/largemultifasta.t ………… ok<br>
   <br> t/AlignIO/maf.t …………………… ok<br>
   <br> t/AlignIO/mase.t ………………….. ok<br>
   <br> t/AlignIO/mega.t ………………….. ok<br>
   <br> t/AlignIO/meme.t ………………….. ok<br>
   <br> t/AlignIO/metafasta.t ……………… ok<br>
   <br> t/AlignIO/msf.t …………………… ok<br>
   <br> t/AlignIO/nexml.t …………………. skipped: The optional module Bio::Phylo (or dependencies thereof) was not installed<br>
   <br> t/AlignIO/nexus.t …………………. ok<br>
   <br> t/AlignIO/pfam.t ………………….. ok<br>
   <br> t/AlignIO/phylip.t ………………… ok<br>
   <br> t/AlignIO/po.t ……………………. ok<br>
   <br> t/AlignIO/prodom.t ………………… ok<br>
   <br> t/AlignIO/psi.t …………………… ok<br>
   <br> t/AlignIO/selex.t …………………. ok<br>
   <br> t/AlignIO/stockholm.t ……………… ok<br>
   <br> t/AlignIO/xmfa.t ………………….. ok<br>
   <br> t/Alphabet.t ……………………… ok<br>
   <br> t/Annotation/Annotation.t ………….. ok<br>
   <br> t/Annotation/AnnotationAdaptor.t ……. ok<br>
   <br> t/Assembly/ContigSpectrum.t ………… skipped: The optional module DB_File (or dependencies thereof) was not installed<br>
   <br> t/Assembly/IO/bowtie.t …………….. skipped: The optional module DB_File (or dependencies thereof) was not installed<br>
   <br> t/Assembly/IO/sam.t ……………….. skipped: The optional module DB_File (or dependencies thereof) was not installed<br>
   <br> t/Assembly/core.t …………………. skipped: The optional module DB_File (or dependencies thereof) was not installed<br>
   <br> t/Cluster/UniGene.t ……………….. ok<br>
   <br> t/ClusterIO/ClusterIO.t ……………. ok<br>
   <br> t/ClusterIO/SequenceFamily.t ……….. ok<br>
   <br> t/ClusterIO/unigene.t ……………… ok<br>
   <br> t/Coordinate/CoordinateBoundaryTest.t .. ok<br>
   <br> t/Coordinate/CoordinateGraph.t ……… ok<br>
   <br> t/Coordinate/CoordinateMapper.t …….. ok<br>
   <br> t/Coordinate/GeneCoordinateMapper.t …. ok<br>
   <br> t/Draw/Pictogram.t ………………… skipped: The optional module SVG (or dependencies thereof) was not installed<br>
   <br> t/LiveSeq/Chain.t …………………. ok<br>
   <br> t/LiveSeq/LiveSeq.t ……………….. ok<br>
   <br> t/LiveSeq/Mutation.t ………………. ok<br>
   <br> t/LiveSeq/Mutator.t ……………….. ok<br>
   <br> t/LocalDB/BioDBGFF.t ………………. 7/275<br>
   <br> You are loading a Bio::DB::GFF database with GFF3 formatted data.<br>
   <br> While this will likely work fine, the Bio::DB::GFF schema does not<br>
   <br> always faithfully capture the complexity represented in GFF3 files.<br>
   <br> Unless you have a specific reason for using Bio::DB::GFF, we suggest<br>
   <br> that you use a Bio::DB::SeqFeature::Store database and its corresponding<br>
   <br> loader, bp_seqfeature_load.pl.
  </div>
<div>
   
  </div>
<div>
   t/LocalDB/BioDBGFF.t ………………. ok<br>
   <br> t/LocalDB/Fasta.t …………………. 1/107 Can’t locate Digest/MD5.pm in @INC (@INC contains: . /root/.cpan/build/BioPerl-1.6.924-4qo47i/blib/lib /root/.cpan/build/BioPerl-1.6.924-4qo47i/blib/arch /root/.cpan/build/BioPerl-1.6.924-4qo47i /root/perl5/lib/perl5/5.16.3/x86_64-linux-thread-multi /root/perl5/lib/perl5/5.16.3 /root/perl5/lib/perl5/x86_64-linux-thread-multi /root/perl5/lib/perl5 /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5) at Bio/DB/IndexedBase.pm line 503.<br>
   <br> # Looks like you planned 107 tests but ran 95.<br>
   <br> # Looks like your test exited with 2 just after 95.<br>
   <br> t/LocalDB/Fasta.t …………………. Dubious, test returned 2 (wstat 512, 0x200)<br>
   <br> Failed 12/107 subtests<br>
   <br>         (less 1 skipped subtest: 94 okay)<br>
   <br> t/LocalDB/Flat.t ………………….. skipped: The optional module DB_File (or dependencies thereof) was not installed<br>
   <br> t/LocalDB/Index/Blast.t ……………. ok<br>
   <br> t/LocalDB/Index/BlastTable.t ……….. ok<br>
   <br> t/LocalDB/Index/Index.t ……………. skipped: The optional module DB_File (or dependencies thereof) was not installed<br>
   <br> t/LocalDB/Qual.t ………………….. ok<br>
   <br> t/LocalDB/Registry.t ………………. ok<br>
   <br> t/LocalDB/SeqFeature.t …………….. skipped: The optional module DB_File (or dependencies thereof) was not installed<br>
   <br> t/LocalDB/Taxonomy/greengenes.t …….. ok<br>
   <br> t/LocalDB/Taxonomy/silva.t …………. ok<br>
   <br> t/LocalDB/transfac_pro.t …………… skipped: The optional module DB_File (or dependencies thereof) was not installed<br>
   <br> t/Map/Cyto.t ……………………… ok<br>
   <br> t/Map/Linkage.t …………………… ok<br>
   <br> t/Map/Map.t ………………………. ok<br>
   <br> t/Map/MapIO.t …………………….. ok<br>
   <br> t/Map/MicrosatelliteMarker.t ……….. ok<br>
   <br> t/Map/Physical.t ………………….. ok<br>
   <br> t/Matrix/IO/masta.t ……………….. ok<br>
   <br> t/Matrix/IO/psm.t …………………. ok<br>
   <br> t/Matrix/InstanceSite.t ……………. ok<br>
   <br> t/Matrix/Matrix.t …………………. ok<br>
   <br> t/Matrix/ProtMatrix.t ……………… ok<br>
   <br> t/Matrix/ProtPsm.t ………………… ok<br>
   <br> t/Matrix/SiteMatrix.t ……………… ok<br>
   <br> t/Ontology/GOterm.t ……………….. skipped: The optional module Graph::Directed (or dependencies thereof) was not installed<br>
   <br> t/Ontology/GraphAdaptor.t ………….. skipped: The optional module Graph (or dependencies thereof) was not installed<br>
   <br> t/Ontology/IO/go.t ………………… skipped: The optional module Graph (or dependencies thereof) was not installed<br>
   <br> t/Ontology/IO/interpro.t …………… skipped: The optional module XML::Parser::PerlSAX (or dependencies thereof) was not installed<br>
   <br> t/Ontology/IO/obo.t ……………….. skipped: The optional module Graph (or dependencies thereof) was not installed<br>
   <br> t/Ontology/Ontology.t ……………… skipped: The optional module Graph (or dependencies thereof) was not installed<br>
   <br> t/Ontology/OntologyEngine.t ………… skipped: The optional module Graph::Directed (or dependencies thereof) was not installed<br>
   <br> t/Ontology/OntologyStore.t …………. skipped: The optional module Graph (or dependencies thereof) was not installed<br>
   <br> t/Ontology/Relationship.t ………….. skipped: The optional module Graph::Directed (or dependencies thereof) was not installed<br>
   <br> t/Ontology/RelationshipType.t ………. skipped: The optional module Graph::Directed (or dependencies thereof) was not installed<br>
   <br> t/Ontology/Term.t …………………. skipped: The optional module Graph::Directed (or dependencies thereof) was not installed<br>
   <br> t/Perl.t …………………………. ok<br>
   <br> t/Phenotype/Correlate.t ……………. ok<br>
   <br> t/Phenotype/MeSH.t ………………… ok<br>
   <br> t/Phenotype/Measure.t ……………… ok<br>
   <br> t/Phenotype/MiniMIMentry.t …………. ok<br>
   <br> t/Phenotype/OMIMentry.t ……………. ok<br>
   <br> t/Phenotype/OMIMentryAllelicVariant.t .. ok<br>
   <br> t/Phenotype/OMIMparser.t …………… ok<br>
   <br> t/Phenotype/Phenotype.t ……………. ok<br>
   <br> t/PodSyntax.t …………………….. skipped: Test::Pod 1.00 required for testing POD<br>
   <br> t/PopGen/Coalescent.t ……………… ok<br>
   <br> t/PopGen/HtSNP.t ………………….. ok<br>
   <br> t/PopGen/MK.t …………………….. ok<br>
   <br> t/PopGen/PopGen.t …………………. ok<br>
   <br> t/PopGen/PopGenSims.t ……………… ok<br>
   <br> t/PopGen/TagHaplotype.t ……………. ok<br>
   <br> t/RemoteDB/BioFetch.t ……………… skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/CUTG.t …………………. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/EMBL.t …………………. skipped: The optional module HTTP::Request::Common (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/EntrezGene.t ……………. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/GenBank.t ………………. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/GenPept.t ………………. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/HIV/HIV.t ………………. skipped: The optional module XML::Simple (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/HIV/HIVAnnotProcessor.t ….. ok<br>
   <br> t/RemoteDB/HIV/HIVQuery.t ………….. skipped: The optional module XML::Simple (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/HIV/HIVQueryHelper.t …….. skipped: The optional module XML::Simple (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/MeSH.t …………………. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/Query/GenBank.t …………. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/RefSeq.t ……………….. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/SeqHound.t ……………… skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/SeqRead_fail.t ………….. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/SeqVersion.t ……………. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/SwissProt.t …………….. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/RemoteDB/Taxonomy.t ……………… skipped: The optional module DB_File (or dependencies thereof) was not installed<br>
   <br> t/Restriction/Analysis-refac.t ……… ok<br>
   <br> t/Restriction/Analysis.t …………… ok<br>
   <br> t/Restriction/Gel.t ……………….. ok<br>
   <br> t/Restriction/IO.t ………………… ok<br>
   <br> t/Root/Exception.t ………………… ok<br>
   <br> t/Root/HTTPget.t ………………….. skipped: Network tests have not been requested<br>
   <br> t/Root/IO.t ………………………. ok<br>
   <br> t/Root/RootI.t ……………………. ok<br>
   <br> t/Root/Storable.t …………………. ok<br>
   <br> t/Root/Utilities.t ………………… 1/56<br>
   <br> ——————— WARNING ———————<br>
   <br> MSG: Not owner of file t/data/test.txt. Compressing to temp file /tmp/DN1geA3UTk.tmp.bioperl.gz.<br>
   <br> —————————————————<br>
   <br> t/Root/Utilities.t ………………… ok<br>
   <br> t/SearchDist.t ……………………. skipped: The optional module Bio::Ext::Align (or dependencies thereof) was not installed<br>
   <br> t/SearchIO/CigarString.t …………… ok<br>
   <br> t/SearchIO/SearchIO.t ……………… ok<br>
   <br> t/SearchIO/SimilarityPair.t ………… ok<br>
   <br> t/SearchIO/Tiling.t ……………….. ok<br>
   <br> t/SearchIO/Writer/GbrowseGFF.t ……… ok<br>
   <br> t/SearchIO/Writer/HSPTableWriter.t ….. ok<br>
   <br> t/SearchIO/Writer/HTMLWriter.t ……… ok<br>
   <br> t/SearchIO/Writer/HitTableWriter.t ….. ok<br>
   <br> t/SearchIO/Writer/TextWriter.t ……… ok<br>
   <br> t/SearchIO/axt.t ………………….. ok<br>
   <br> t/SearchIO/blast.t ………………… ok<br>
   <br> t/SearchIO/blast_pull.t ……………. ok<br>
   <br> t/SearchIO/blasttable.t ……………. ok<br>
   <br> t/SearchIO/blastxml.t ……………… skipped: The optional module XML::SAX (or dependencies thereof) was not installed<br>
   <br> t/SearchIO/cross_match.t …………… ok<br>
   <br> t/SearchIO/erpin.t ………………… ok<br>
   <br> t/SearchIO/exonerate.t …………….. ok<br>
   <br> t/SearchIO/fasta.t ………………… ok<br>
   <br> t/SearchIO/gmap_f9.t ………………. ok<br>
   <br> t/SearchIO/hmmer.t ………………… ok<br>
   <br> t/SearchIO/hmmer_pull.t ……………. ok<br>
   <br> t/SearchIO/infernal.t ……………… ok<br>
   <br> t/SearchIO/megablast.t …………….. ok<br>
   <br> t/SearchIO/psl.t ………………….. ok<br>
   <br> t/SearchIO/rnamotif.t ……………… ok<br>
   <br> t/SearchIO/sim4.t …………………. ok<br>
   <br> t/SearchIO/waba.t …………………. ok<br>
   <br> t/SearchIO/wise.t …………………. ok<br>
   <br> t/Seq/DBLink.t ……………………. ok<br>
   <br> t/Seq/EncodedSeq.t ………………… ok<br>
   <br> t/Seq/LargeLocatableSeq.t ………….. ok<br>
   <br> t/Seq/LargePSeq.t …………………. ok<br>
   <br> t/Seq/LocatableSeq.t ………………. ok<br>
   <br> t/Seq/MetaSeq.t …………………… ok<br>
   <br> t/Seq/PrimaryQual.t ……………….. ok<br>
   <br> t/Seq/PrimarySeq.t ………………… ok<br>
   <br> t/Seq/PrimedSeq.t …………………. ok<br>
   <br> t/Seq/Quality.t …………………… ok<br>
   <br> t/Seq/Seq.t ………………………. ok<br>
   <br> t/Seq/SimulatedRead.t ……………… ok<br>
   <br> t/Seq/WithQuality.t ……………….. ok<br>
   <br> t/SeqEvolution.t ………………….. ok<br>
   <br> t/SeqFeature/Amplicon.t ……………. ok<br>
   <br> t/SeqFeature/Clone.t ………………. ok<br>
   <br> t/SeqFeature/Collection.t ………….. skipped: The optional module DB_File (or dependencies thereof) was not installed<br>
   <br> t/SeqFeature/Computation.t …………. ok<br>
   <br> t/SeqFeature/FeaturePair.t …………. ok<br>
   <br> t/SeqFeature/Gene.t ……………….. ok<br>
   <br> t/SeqFeature/Generic.t …………….. ok<br>
   <br> t/SeqFeature/Location.t ……………. ok<br>
   <br> t/SeqFeature/LocationFactory.t ……… ok<br>
   <br> t/SeqFeature/Primer.t ……………… ok<br>
   <br> t/SeqFeature/Range.t ………………. ok<br>
   <br> t/SeqFeature/RangeI.t ……………… ok<br>
   <br> t/SeqFeature/SeqAnalysisParser.t ……. ok<br>
   <br> t/SeqFeature/SubSeq.t ……………… ok<br>
   <br> t/SeqFeature/Unflattener.t …………. ok<br>
   <br> t/SeqIO/Handler.t …………………. ok<br>
   <br> t/SeqIO/MultiFile.t ……………….. ok<br>
   <br> t/SeqIO/Multiple_fasta.t …………… ok<br>
   <br> t/SeqIO/SeqBuilder.t ………………. ok<br>
   <br> t/SeqIO/SeqIO.t …………………… ok<br>
   <br> t/SeqIO/Splicedseq.t ………………. 1/19 Can’t locate LWP/UserAgent.pm in @INC (@INC contains: . /root/.cpan/build/BioPerl-1.6.924-4qo47i/blib/lib /root/.cpan/build/BioPerl-1.6.924-4qo47i/blib/arch /root/.cpan/build/BioPerl-1.6.924-4qo47i /root/perl5/lib/perl5/5.16.3/x86_64-linux-thread-multi /root/perl5/lib/perl5/5.16.3 /root/perl5/lib/perl5/x86_64-linux-thread-multi /root/perl5/lib/perl5 /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5) at Bio/DB/Query/WebQuery.pm line 85.<br>
   <br> BEGIN failed–compilation aborted at Bio/DB/Query/WebQuery.pm line 85.<br>
   <br> Compilation failed in require at (eval 59) line 2.<br>
   <br>         …propagated at /usr/share/perl5/base.pm line 84.<br>
   <br> BEGIN failed–compilation aborted at Bio/DB/Query/GenBank.pm line 115.<br>
   <br> Compilation failed in require at Bio/DB/NCBIHelper.pm line 90.<br>
   <br> BEGIN failed–compilation aborted at Bio/DB/NCBIHelper.pm line 90.<br>
   <br> Compilation failed in require at (eval 58) line 2.<br>
   <br>         …propagated at /usr/share/perl5/base.pm line 84.<br>
   <br> BEGIN failed–compilation aborted at Bio/DB/GenBank.pm line 171.<br>
   <br> Compilation failed in require at t/SeqIO/Splicedseq.t line 74.<br>
   <br> BEGIN failed–compilation aborted at t/SeqIO/Splicedseq.t line 74.<br>
   <br> # Looks like you planned 19 tests but ran 1.<br>
   <br> # Looks like your test exited with 2 just after 1.<br>
   <br> t/SeqIO/Splicedseq.t ………………. Dubious, test returned 2 (wstat 512, 0x200)<br>
   <br> Failed 18/19 subtests<br>
   <br> t/SeqIO/abi.t …………………….. skipped: The optional module Bio::SeqIO::staden::read (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/ace.t …………………….. ok<br>
   <br> t/SeqIO/agave.t …………………… skipped: The optional module XML::Writer (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/alf.t …………………….. skipped: The optional module Bio::SeqIO::staden::read (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/asciitree.t ……………….. ok<br>
   <br> t/SeqIO/bsml.t ……………………. skipped: The optional module XML::DOM (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/bsml_sax.t ………………… skipped: The optional module XML::SAX (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/chadoxml.t ………………… skipped: The optional module XML::Writer (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/chaos.t …………………… ok<br>
   <br> t/SeqIO/chaosxml.t ………………… ok<br>
   <br> t/SeqIO/ctf.t …………………….. skipped: The optional module Bio::SeqIO::staden::read (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/embl.t ……………………. ok<br>
   <br> t/SeqIO/entrezgene.t ………………. skipped: The optional module Bio::ASN1::EntrezGene (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/excel.t …………………… skipped: The optional module Spreadsheet::ParseExcel (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/exp.t …………………….. skipped: The optional module Bio::SeqIO::staden::read (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/fasta.t …………………… ok<br>
   <br> t/SeqIO/fastq.t …………………… ok<br>
   <br> t/SeqIO/flybase_chadoxml.t …………. skipped: The optional module XML::Writer (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/game.t ……………………. skipped: The optional module XML::Parser::PerlSAX (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/gbxml.t …………………… skipped: The optional module XML::SAX (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/gcg.t …………………….. ok<br>
   <br> t/SeqIO/genbank.t …………………. ok<br>
   <br> t/SeqIO/interpro.t ………………… skipped: The optional module XML::DOM::XPath (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/kegg.t ……………………. ok<br>
   <br> t/SeqIO/largefasta.t ………………. ok<br>
   <br> t/SeqIO/lasergene.t ……………….. ok<br>
   <br> t/SeqIO/locuslink.t ……………….. skipped: The optional module Graph::Directed (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/mbsout.t ………………….. ok<br>
   <br> t/SeqIO/metafasta.t ……………….. ok<br>
   <br> t/SeqIO/msout.t …………………… ok<br>
   <br> t/SeqIO/nexml.t …………………… skipped: The optional module Bio::Phylo (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/phd.t …………………….. ok<br>
   <br> t/SeqIO/pir.t …………………….. ok<br>
   <br> t/SeqIO/pln.t …………………….. skipped: The optional module Bio::SeqIO::staden::read (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/qual.t ……………………. ok<br>
   <br> t/SeqIO/raw.t …………………….. ok<br>
   <br> t/SeqIO/scf.t …………………….. ok<br>
   <br> t/SeqIO/seqxml.t ………………….. skipped: The optional module XML::LibXML (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/strider.t …………………. skipped: The optional module Convert::Binary::C (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/swiss.t …………………… ok<br>
   <br> t/SeqIO/tab.t …………………….. ok<br>
   <br> t/SeqIO/table.t …………………… skipped: The optional module IO::Scalar (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/tigr.t ……………………. ok<br>
   <br> t/SeqIO/tigrxml.t …………………. skipped: The optional module XML::SAX (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/tinyseq.t …………………. skipped: The optional module XML::Parser::PerlSAX (or dependencies thereof) was not installed<br>
   <br> t/SeqIO/ztr.t …………………….. skipped: The optional module Bio::SeqIO::staden::read (or dependencies thereof) was not installed<br>
   <br> t/SeqTools/Backtranslate.t …………. skipped: The optional module List::MoreUtils (or dependencies thereof) was not installed<br>
   <br> t/SeqTools/CodonTable.t ……………. ok<br>
   <br> t/SeqTools/ECnumber.t ……………… ok<br>
   <br> t/SeqTools/GuessSeqFormat.t ………… ok<br>
   <br> t/SeqTools/OddCodes.t ……………… ok<br>
   <br> t/SeqTools/SeqPattern.t ……………. ok<br>
   <br> t/SeqTools/SeqStats.t ……………… ok<br>
   <br> t/SeqTools/SeqUtils.t ……………… ok<br>
   <br> t/SeqTools/SeqWords.t ……………… ok<br>
   <br> t/Species.t ………………………. ok<br>
   <br> t/Structure/IO.t ………………….. ok<br>
   <br> t/Structure/Structure.t ……………. ok<br>
   <br> t/Symbol.t ……………………….. ok<br>
   <br> t/TaxonTree.t …………………….. skipped: These modules are now probably deprecated<br>
   <br> t/Tools/Alignment/Consed.t …………. ok<br>
   <br> t/Tools/AmpliconSearch.t …………… ok<br>
   <br> t/Tools/Analysis/DNA/ESEfinder.t ……. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/Tools/Analysis/Protein/Domcut.t …… skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/Tools/Analysis/Protein/ELM.t ……… skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/Tools/Analysis/Protein/GOR4.t …….. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/Tools/Analysis/Protein/HNN.t ……… skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/Tools/Analysis/Protein/Mitoprot.t …. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/Tools/Analysis/Protein/NetPhos.t ….. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/Tools/Analysis/Protein/Scansite.t …. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/Tools/Analysis/Protein/Sopma.t ……. skipped: The optional module LWP::UserAgent (or dependencies thereof) was not installed<br>
   <br> t/Tools/EMBOSS/Palindrome.t ………… ok<br>
   <br> t/Tools/Est2Genome.t ………………. ok<br>
   <br> t/Tools/FootPrinter.t ……………… ok<br>
   <br> t/Tools/GFF.t …………………….. ok<br>
   <br> t/Tools/Geneid.t ………………….. ok<br>
   <br> t/Tools/Genewise.t ………………… ok<br>
   <br> t/Tools/Genomewise.t ………………. ok<br>
   <br> t/Tools/Genpred.t …………………. ok<br>
   <br> t/Tools/Hmmer.t …………………… ok<br>
   <br> t/Tools/IUPAC.t …………………… ok<br>
   <br> t/Tools/Lucy.t ……………………. ok<br>
   <br> t/Tools/Match.t …………………… ok<br>
   <br> t/Tools/Phylo/Gerp.t ………………. ok<br>
   <br> t/Tools/Phylo/Molphy.t …………….. ok<br>
   <br> t/Tools/Phylo/PAML.t ………………. ok<br>
   <br> t/Tools/Phylo/Phylip/ProtDist.t …….. ok<br>
   <br> t/Tools/Primer3.t …………………. ok<br>
   <br> t/Tools/Promoterwise.t …………….. ok<br>
   <br> t/Tools/Pseudowise.t ………………. ok<br>
   <br> t/Tools/QRNA.t ……………………. ok<br>
   <br> t/Tools/RandDistFunctions.t ………… ok<br>
   <br> t/Tools/RepeatMasker.t …………….. ok<br>
   <br> t/Tools/Run/RemoteBlast.t ………….. skipped: The optional module LWP (or dependencies thereof) was not installed<br>
   <br> t/Tools/Run/RemoteBlast_rpsblast.t ….. skipped: The optional module LWP (or dependencies thereof) was not installed<br>
   <br> t/Tools/Run/StandAloneBlast.t ………. ok<br>
   <br> t/Tools/Run/WBCommandExts.t ………… skipped: The optional module Bio::Tools::Run::WrapperBase::CommandExts (or dependencies thereof) was not installed<br>
   <br> t/Tools/Run/WrapperBase.t ………….. ok<br>
   <br> t/Tools/Seg.t …………………….. ok<br>
   <br> t/Tools/SiRNA.t …………………… ok<br>
   <br> t/Tools/Sigcleave.t ……………….. ok<br>
   <br> t/Tools/Signalp.t …………………. ok<br>
   <br> t/Tools/Signalp/ExtendedSignalp.t …… ok<br>
   <br> t/Tools/Sim4.t ……………………. ok<br>
   <br> t/Tools/Spidey/Spidey.t ……………. ok<br>
   <br> t/Tools/TandemRepeatsFinder.t ………. ok<br>
   <br> t/Tools/TargetP.t …………………. ok<br>
   <br> t/Tools/Tmhmm.t …………………… ok<br>
   <br> t/Tools/ePCR.t ……………………. ok<br>
   <br> t/Tools/pICalculator.t …………….. ok<br>
   <br> t/Tools/rnamotif.t ………………… skipped: These modules are now probably deprecated<br>
   <br> t/Tools/tRNAscanSE.t ………………. ok<br>
   <br> t/Tree/Compatible.t ……………….. skipped: The optional module Set::Scalar (or dependencies thereof) was not installed<br>
   <br> t/Tree/Node.t …………………….. ok<br>
   <br> t/Tree/PhyloNetwork/Factory.t ………. skipped: The optional module Bio::PhyloNetwork (or dependencies thereof) was not installed<br>
   <br> t/Tree/PhyloNetwork/GraphViz.t ……… skipped: The optional module Bio::PhyloNetwork (or dependencies thereof) was not installed<br>
   <br> t/Tree/PhyloNetwork/MuVector.t ……… ok<br>
   <br> t/Tree/PhyloNetwork/PhyloNetwork.t ….. skipped: The optional module Graph::Directed (or dependencies thereof) was not installed<br>
   <br> t/Tree/PhyloNetwork/RandomFactory.t …. skipped: The optional module Bio::PhyloNetwork (or dependencies thereof) was not installed<br>
   <br> t/Tree/PhyloNetwork/TreeFactory.t …… skipped: The optional module Bio::PhyloNetwork (or dependencies thereof) was not installed<br>
   <br> t/Tree/RandomTreeFactory.t …………. ok<br>
   <br> t/Tree/Tree.t …………………….. ok<br>
   <br> t/Tree/TreeIO.t …………………… ok<br>
   <br> t/Tree/TreeIO/lintree.t ……………. ok<br>
   <br> t/Tree/TreeIO/newick.t …………….. ok<br>
   <br> t/Tree/TreeIO/nexml.t ……………… skipped: The optional module Bio::Phylo (or dependencies thereof) was not installed<br>
   <br> t/Tree/TreeIO/nexus.t ……………… ok<br>
   <br> t/Tree/TreeIO/nhx.t ……………….. ok<br>
   <br> t/Tree/TreeIO/phyloxml.t …………… skipped: The optional module XML::LibXML (or dependencies thereof) was not installed<br>
   <br> t/Tree/TreeIO/svggraph.t …………… ok<br>
   <br> t/Tree/TreeIO/tabtree.t ……………. ok<br>
   <br> t/Tree/TreeStatistics.t ……………. ok<br>
   <br> t/Variation/AAChange.t …………….. ok<br>
   <br> t/Variation/AAReverseMutate.t ………. ok<br>
   <br> t/Variation/Allele.t ………………. ok<br>
   <br> t/Variation/DNAMutation.t ………….. ok<br>
   <br> t/Variation/RNAChange.t ……………. ok<br>
   <br> t/Variation/SNP.t …………………. ok<br>
   <br> t/Variation/SeqDiff.t ……………… ok<br>
   <br> t/Variation/Variation_IO.t …………. skipped: The optional module XML::Writer (or dependencies thereof) was not installed<br>
   <br> t/nexml.t ………………………… skipped: The optional module Bio::Phylo (or dependencies thereof) was not installed
  </div>
<div>
   
  </div>
<div>
   Test Summary Report<br>
   <br> ——————-<br>
   <br> t/LocalDB/Fasta.t                    (Wstat: 512 Tests: 95 Failed: 0)<br>
   <br>   Non-zero exit status: 2<br>
   <br>   Parse errors: Bad plan.  You planned 107 tests but ran 95.<br>
   <br> t/SeqIO/Splicedseq.t                 (Wstat: 512 Tests: 1 Failed: 0)<br>
   <br>   Non-zero exit status: 2<br>
   <br>   Parse errors: Bad plan.  You planned 19 tests but ran 1.<br>
   <br> Files=334, Tests=17147, 94 wallclock secs ( 2.98 usr  0.67 sys + 86.88 cusr  6.43 csys = 96.96 CPU)<br>
   <br> Result: FAIL #测试结果是失败的
  </div>
<p> Failed 2/334 test programs. 0/17147 subtests failed.<br>
  <br>   CJFIELDS/BioPerl-1.6.924.tar.gz<br>
  <br>   ./Build test — NOT OK<br>
  <br> //hint// to see the cpan-testers results for installing this module, try:<br>
  <br>   reports CJFIELDS/BioPerl-1.6.924.tar.gz<br>
  <br> Failed during this command:<br>
  <br>  CJFIELDS/BioPerl-1.6.924.tar.gz              : make_test NO<br>
  </p>
</div>
</div>
	</div>

	<footer class="amp-wp-article-footer">
			<div class="amp-wp-meta amp-wp-tax-category">
		Categories: <a href="https://uzzz.org/category/wangluoanquan/" rel="category tag">网络安全</a>	</div>

	</footer>
</article>

<footer class="amp-wp-footer">
	<div>
		<h2>有组织在!</h2>
		<a href="#top" class="back-to-top">Back to top</a>
	</div>
</footer>




</body></html>
